pkgs/top-level/python-packages.nix

```diff
@@ -12901,6 +12899,25 @@ in modules // {
     };
   };
 
+  emoji = buildPythonPackage rec {
+    name = "emoji-${version}";
+    version = "0.3.9";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/e/emoji/${name}.tar.gz";
+      sha256 = "19p5c2nlq0w9972rf9fghyswnijwgig5f8cyzs32kppnmzvzbkxw";
+    };
+
+    buildInputs = with self; [ nose ];
```

Comment:
checkPhase doesn't explicitly run nose but instead runs `python setup.py test`. It is possible that nose is specified as test suite in setup.py in which case it is *supposed* to run. But well, with Python packages you're never quite sure what is going to happen. Therefore, it's best to check whether you see that a certain amount of tests have run or not.